source/_integrations/elevenlabs.markdown

```diff
@@ -0,0 +1,42 @@
+---
+title: ElevenLabs text-to-speech
+description: Instructions on how to setup ElevenLabs text-to-speech with Home Assistant.
+ha_category:
+  - Text-to-speech
+ha_release: 2024.4.3
+ha_iot_class: Cloud Push
+ha_domain: elevenlabstts
+ha_platforms:
+  - tts
+ha_config_flow: true
+ha_integration_type: integration
+---
+
+The `elevenlabstts` text-to-speech platform uses the official [ElevenLabs SDK](https://github.com/elevenlabs/elevenlabs-python) to read a text with natural sounding voices.
```

Comment:
Not sure why it's relevant it's using the official SDK

Reply:
Should I just remove the whole line then?